examples/mobile/HomeApp/webclip/video-service/webclip.html

```diff
@@ -0,0 +1,60 @@
+<div class="ui-card ui-card-service" class="app-video-control">
+	<link rel="stylesheet" href="css/style.css" />
+	<div class="ui-content-subheader">
+		Video Collection
+	</div>
+	<div class="ui-content-area">
+		<div class="ui-section-changer">
+			<div class="ui-section-changer-container">
+				<section>
```

Comment:
Is it needed  class="ui-content-area" like <section class="ui-content-area">?

Reply:
You are right.

Reply:
Done.